CRM/Dedupe/BAO/DedupeRuleGroup.php

```diff
@@ -131,6 +138,36 @@ public function tableDropQuery() {
     return 'DROP TEMPORARY TABLE IF EXISTS dedupe';
   }
 
+  /**
+   * Find existing duplicates in the database for the given rule and limitations.
+   *
+   * @return void
+   */
+  public static function hook_civicrm_findExistingDuplicates(\Civi\Core\Event\GenericHookEvent $event) {
```

Comment:
this should change to self_hook_civicrm_findExistingDuplicates as per https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/BAO/OptionGroup.php#L83

Reply:
I think what @eileenmcnaughton did is correct. Since she needs to specify priority this is the way to do it.